layouts/partials/head.html

```diff
@@ -156,4 +156,6 @@
             {{ errorf "Blog posts require tags: %s" .File.Path }}
         {{ end }}
     {{ end }}
+
+    <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
We'll want to remove this and reposition it after the element, as suggested by the Hugo implementation (and the [Twitter docs](https://developer.twitter.com/en/docs/twitter-for-websites/embedded-tweets/guides/cms-best-practices) as well):

```
<blockquote class="twitter-tweet">
...
</blockquote>
<script async src="https://platform.twitter.com/widgets.js"></script>
```

Reply:
👍